lib/developer_portal/lib/liquid/filters/rails_helpers.rb

```diff
@@ -268,6 +270,16 @@ def controller
       def button(title, url, method, options={})
         view.button_to title.to_s, url.to_s, {method: method}.merge(options)
       end
+
+      ENABLE_API_DOCS_PROXY = "window.enableApiDocsProxy = (typeof window.enableApiDocsProxy !== undefined ? window.enableApiDocsProxy : #{Rails.configuration.three_scale.active_docs_proxy_disabled.blank?});\n"
```

Comment:
```suggestion
      ENABLE_API_DOCS_PROXY = "window.enableApiDocsProxy = window.enableApiDocsProxy === undefined && #{Rails.configuration.three_scale.active_docs_proxy_disabled.blank?});\n"
```

Reply:
wait ... this wouldn't put it to `false` if it is ever defined?